website/pages/en/index.js

```diff
@@ -293,12 +293,19 @@ const UsersSection = ({ language }) => {
             </a>
             <div style={{ marginLeft: ".7em", width: "260px" }}>
               <p>
-                Regularly used by more than <strong>70%</strong> of respondents
-                to State of JS 2020
+                Regularly used by: <br />
+                <a href="https://2021.stateofjs.com/en-US/other-tools/#utilities">
+                  <strong>
+                    More than 83% of respondents to State of JS 2021.
+                  </strong>
+                </a>
+                <br />
+                <a href="https://2020.stateofjs.com/en-US/other-tools/#utilities">
+                  <strong>
+                    More than 70% of respondents to State of JS 2020.
+                  </strong>
+                </a>
```

Comment:
```suggestion
<ul>
  <li></li>
  <li></li>
```

?

Remove `<strong>` please

Reply:
I suggest to use `<ul>` element here, why are you ignoring it without any reason?